### PR TITLE
Update `Work directly with JSON` BBE MD File

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Update `Work directly with JSON` BBE MD file.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/working-directly-with-json/working_directly_with_json.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
-# Working directly with JSON
+# Work directly with JSON
```
Comment (MaryamZi) [inline examples/working-directly-with-json/working_directly_with_json.md:1]: We have multiple BBEs that have similar titles. E.g., [Ignoring return values and errors](https://ballerina.io/learn/by-example/ignoring-return-values-and-errors.html), [Controlling openness](https://ballerina.io/learn/by-example/controlling-openness.html), etc. Can we leave these as is and do the changes in a separate PR if required?

File: examples/working-directly-with-json/working_directly_with_json.out [hunk 2]
```diff
@@ -1,3 +1,3 @@
-bal run working_directly_with_json.bal
+$ bal run working_directly_with_json.bal
```
Comment (MaryamZi) [inline examples/working-directly-with-json/working_directly_with_json.out:1]: `$` was removed after some discussions IIRC. Shall we finalize if we are adding it back before changing it here?

Comment (praneesha) [reply examples/working-directly-with-json/working_directly_with_json.out:1]: @anupama-pathirage @sameerajayasoma

File: examples/working-directly-with-json/working_directly_with_json.md (hunk 1 above)
Comment (praneesha) [reply examples/working-directly-with-json/working_directly_with_json.md]: Fixed.

File: examples/working-directly-with-json/working_directly_with_json.out (hunk 2 above)
Comment (MaryamZi) [reply examples/working-directly-with-json/working_directly_with_json.out:1]: I prefer having it too btw. And as discussed, since with BBEs the user may not generally have the same file name, it's highly unlikely they'll copy the command as is. So copying shouldn't be an issue here IMO. We'll also have the output in the same file which is probably why we don't give the option to copy atm?

Comment (anupama-pathirage) [reply examples/working-directly-with-json/working_directly_with_json.out:1]: What if we add it later, when we figure out things (like having a way to copy without the `$` sign etc)

Comment (praneesha) [reply examples/working-directly-with-json/working_directly_with_json.out:1]: Thanks, Yasith! Will keep the $sign as we will be implementing this fix soon.

**Review by anupama-pathirage — APPROVED** [review]

**Review by MaryamZi — APPROVED** [review]

File: examples/working-directly-with-json/working_directly_with_json.out (hunk 2 above)
Comment (MaryamZi) [reply examples/working-directly-with-json/working_directly_with_json.out:1]: I was just thinking maybe @keizer619's comment about reverting (deleting the description file) in a single PR would apply here too? Should we do this change for all BBEs also in a single PR?

Comment (praneesha) [reply examples/working-directly-with-json/working_directly_with_json.out:1]: We have already merged the fix of this and now the $sign isn't getting copied. Therefore, we can finalize adding this.
